scripts/r.import/r.import.py

```diff
@@ -156,8 +166,37 @@ def is_projection_matching(GDALdatasource):
         return False
 
 
+def createvrt(dst, src, nodata=None, region=None, bandlist=None, s_crs=None):
+    # Lazy import
+    from osgeo import gdal
```

Comment:
This import is failing in tests, but looks like the correct syntax, `import gdal` was deprecated at one point.